x-pack/test/functional/apps/lens/group2/dashboard.ts

```diff
@@ -70,7 +70,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await dashboardAddPanel.closeAddPanel();
       await PageObjects.lens.goToTimeRange();
       await retry.try(async () => {
-        await clickInChart(6, 5); // hardcoded position of bar, depends heavy on data and charts implementation
+        await clickInChart(30, 5); // hardcoded position of bar, depends heavy on data and charts implementation
```

Comment:
I haven't tested it yet but @markov00 why do we need to update this test? What has changed?

Reply:
Sorry I thought I've described that in a comment but apparently no.
This PR https://github.com/elastic/elastic-charts/pull/1906 fixed a unnecessary whitespace applied to multi-layer time axis. Removing this white space, the chart is now a bit more wider and the bars are now slightly repositioned.
In this test the bar to click was positioned ~5px to the right from the  center of the canvas ( :( yes that's is how it works in our functional test). With this white space removal, the chart gained about 25/30 pixel in horizontal space, and the bars are actually resized and repositioned a bit along the horizontal space.

Reply:
Aha, got it Marco! Will review!